VideoApp/VideoApp/Stores/Auth/InternalAuthStore.swift

```diff
@@ -117,7 +117,7 @@ private extension TwilioEnvironment {
         switch self {
         case .production: return "twilio-video-react.appspot.com"
         case .staging: return "stage-dot-twilio-video-react.appspot.com"
-        case .development: return "dev-dot-twilio-video-react.appspot.com"
+        case .development: return "large-rooms-dev-dot-twilio-video-react.appspot.com"
```

Comment:
To test the app, switch environment to `development` in settings > internal > environment.

Reply:
Also this will need to be reverted before merging to main branch.